src/import-data/useImportRequest.test.ts

```diff
@@ -127,6 +151,18 @@ describe('getImportRequest', () => {
   ];
 
   beforeAll(() => {
+    const mockDataRepo = {
```

Comment:
TOL (non-blocking): I wonder if we should think about having some sort of default mocking that tests can use for each ajax file, maintained next to the ajax code. That way we don't need to repeat these object creations and instead can use the default (unless you want to explicitly use non-default values)

Reply:
I think the Ajax stuff may be in flux for a while as we break up the main Ajax module and figure out patterns for the future, so I'm not sure how much I'd want to invest in that now.

Generally though, I'd definitely like API requests to be easier to mock. This one was a pain to type. I've also found the shared workspace fixtures to be a big time saver, and it would be nice to have similar fixtures for other API responses.